src/components/Body.tsx

```diff
@@ -9,7 +8,7 @@ const Body: React.FC<BodyProps> = ({ children, github_edit_url }) => {
             className="github-edit"
             href={github_edit_url}
           >
-            <img src={PREFIX + "/svg/github.svg"} />
+            <img src={"/workers/svg/github.svg"} />
```

Comment:
Since prefix is being hardcoded, I'd prefer to keep the `PREFIX` constant. Then in `./utils`, `PREFIX = "/workers"` that way if we change this in the future its more simple than finding all `/workers` references. 

Buuut this works and that makes me happy to merge that is more work than worth!

Reply:
yea that works for me!